cookbook/metadata.rb

```diff
@@ -9,6 +9,9 @@
 description      "Installs/Configures a berkshelf-api server"
 long_description "Installs/Configures a berkshelf-api server"
 version          Berkshelf::API::VERSION
+source_url       'https://github.com/berkshelf/berkshelf-api'
+issues_url       'https://github.com/berkshelf/berkshelf-api/issues'
```

Comment:
You don't know what the target version is? If you didn't guard source_url it's 12 now.